### PR TITLE
Added ability to search API logs

### PR DESCRIPTION
**Pull request by jsma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -438,6 +438,27 @@ Creating a search key that will only search over the body field.
 {'source_engines': ['source-engine-1', 'source-engine-2'], 'type': 'meta', 'name': 'my-meta-engine'}
 ```
 
+### Search the API logs
+
+```python
+>>> client.search_api_logs('my-meta-engine', {
```
Comment (JasonStoltz) [inline README.md:444]: Could you change to this `get_api_logs` just to be consistent with other clients:
https://github.com/elastic/app-search-php/blob/master/resources/api/api-spec.yml#L838
https://github.com/elastic/app-search-php

File: README.md [hunk 2]
```diff
@@ -438,6 +438,27 @@ Creating a search key that will only search over the body field.
...
+  "filters": {
+    "date": {
+      "from": "2020-03-30T00:00:00+00:00",
+      "to": "2020-03-31T00:00:00+00:00"
+    },
+    "status": "429",
+  }
+})
+{'results': [],
```
Comment (JasonStoltz) [inline README.md:453]: Could you update the formatting here to match the formatting of other responses in this README?

Comment (jsma) [reply README.md:453]: Sure, I don't recall if that was the copied/pasted output or where that formatting came from. Not be facetious at all, but I'm not sure _which_ format you prefer. I see examples of "pretty" formatted output, one-liners, and one-liners with elided output:

Screen Shot 2020-07-29 at 9 39 18 AM

Comment (JasonStoltz) [reply README.md:453]: You have a great point. How about "pretty"?

**Comment by JasonStoltz** [comment]
Thanks again for adding this. I'm going to merge and include in a patch release, 7.8.1, shortly.

**Pull request merged**